lips/lip-8.md

```diff
@@ -17,14 +17,22 @@ An extension of the Off-Chain protocol to provide support for more advanced merc
 
 Version 0 of the Off-Chain Protocol is described in [LIP 1](https://lip.libra.org/lip-1/).  Version 1 as described here is an extension of the Off-Chain Protocol and adds features to support more advanced functionality - particularly targeted to support merchant use-cases.  This is inclusive of pull payments, recurring payments, and auth/capture flows.
 
+---
+# Disclaimer
+---
+
+This LIP does not contain the initial phase of a sub-account discovery that is required to start negotiating merchant scenarios.
+The process described below starts at the phase that both sides (particularly the biller side) have the relevant subaddresses. 
+For now, we will assume that such a discovery process took place and the merchant have the buyer subaddress at hand.
```

Comment:
```suggestion
For now, we will assume that such a discovery process took place and the merchant has the buyer subaddress at hand.
```

Reply:
I think the normal approach would be to describe a general strawman flow, then explicitly state what is assumed (e.g., that there is a user -> merchant identity exchange process). As I noted in another PR on DIP-8, there's so much boiler plate before getting into what this DIP does, that it ... isn't particularly accessible. You calling this out, means you see it too :).